### PR TITLE
Add Knuff

### PR DESCRIPTION
**Pull request by dpcrook**
0. [X] Project URL: https://github.com/KnuffApp/Knuff
1. [X] Why should this project be added: macOS app that is useful for debugging and testing iOS push notifications. Even includes a free iOS companion app, available for free in App Store.
2. [X] End all descriptions with a full stop/period
3. [X] Entry is ordered alphabetically
4. [X] Appropriate icon(s) added if applicable (OSS, freeware)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,7 @@
 - [ImageOptim](https://imageoptim.com/mac) - Makes images take up less disk space and load faster. [![Open-Source Software][OSS Icon]](https://github.com/ImageOptim/ImageOptim) ![Freeware][Freeware Icon]
 - [Integrity](http://peacockmedia.software/mac/integrity/free.html) - Easily find your website's broken links. ![Freeware][Freeware Icon]
 - [Kaleidoscope](http://www.kaleidoscopeapp.com/) - Powerful diff and merge application supporting text, images, and folders.
+- [Knuff](https://github.com/KnuffApp/Knuff) - The debug application for Apple Push Notification Service (APNs). [![Open-Source Software][OSS Icon]](https://github.com/ImageOptim/ImageOptim) ![Freeware][Freeware Icon]
```
Comment (iCHAIT) [inline README.md]: I think the OSS icon points to a different source?

**Comment by dpcrook** [comment]
doh! lemme fix that.

File: README.md (hunk 1 above)
Comment (dpcrook) [inline README.md]: it does unintentionally.  lemme fix that.

Comment (dpcrook) [inline README.md]: push with fix made

**Comment by dkhamsing** [comment]
thx

**Pull request merged**